x-pack/plugins/ml/public/application/components/model_snapshots/revert_model_snapshot_flyout/chart_loader.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mlResultsService } from '../../../services/results_service';
+import { CombinedJobWithStats } from '../../../../../common/types/anomaly_detection_jobs';
+import { getSeverityType } from '../../../../../common/util/anomaly_utils';
+import { Anomaly } from '../../../jobs/new_job/common/results_loader/results_loader';
+import { LineChartPoint } from '../../../jobs/new_job/common/chart_loader/chart_loader';
+
+export async function loadEventRateForJob(
```

Comment:
can you please create a service with dependency on `mlResultsService`?

Reply:
Changed in ac22c8ce86162fd9d4db4687850de4ae33873f4b